### PR TITLE
User context docs update

### PR DESCRIPTION
**Pull request by decembrya**

### CODE REVIEW

File: docs2/site/docs/getting-started/user-context.md [hunk 1]
```diff
@@ -28,3 +28,20 @@ public class Query : ObjectGraphType
   }
 }
 ```
+
+If you need to access the User from the http request and populate this globally, you will need to add a User property to your custom `UserContext` class and modify your Startup configuration as follows:
```
Comment (sungam3r) [inline docs2/site/docs/getting-started/user-context.md:32]: > populate this globally

What does _globally_ mean?

Comment (sungam3r) [inline docs2/site/docs/getting-started/user-context.md:32]: > you will need to add a User property

or just
```
var context = new MyGraphQLUserContext(); // or even Dictionary<string, object?>();
context["User"] = user;
```

Comment (sungam3r) [reply docs2/site/docs/getting-started/user-context.md:32]: So I would change wording from _will need to_ to _may_.

File: docs2/site/docs/getting-started/user-context.md [hunk 2]
```diff
@@ -28,3 +28,20 @@ public class Query : ObjectGraphType
...
+      public ClaimsPrincipal User { get; set; }
+
+      public MyGraphQLUserContext(ClaimsPrincipal user)
+      {
+          User = user;
+      }
+  }
+
+  services.AddGraphQL()
+          .AddUserContextBuilder(httpContext => new MyGraphQLUserContext(httpContext.User));
```
Comment (sungam3r) [inline docs2/site/docs/getting-started/user-context.md:46]: There is no such method in this repo. It's method from server project. Worth to note explicitly.

File: docs2/site/docs/getting-started/user-context.md (hunk 1 above)
Comment (decembrya) [reply docs2/site/docs/getting-started/user-context.md]: Thank you for catching this, adjusted the wording and added the other changes suggested.

**Review by sungam3r — APPROVED** [review]

**Comment by sungam3r** [comment]
@Shane32 Merge if OK.

**Pull request merged**